Smoother scrolling with VM browser

DaveVoyles opened the pull request:
HUGE improvement in browser scrolling with the VM after enabling this. Extremely difficult to copy + paste from CNTK tutorials on GitHub and into Jupyter notebooks when scrolling is so slow.

azurecla commented:
Hi __@lucaslin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

DaveVoyles committed:
Smoother scrolling with VM browser

HUGE improvement in browser scrolling with the VM after enabling this. Extremely difficult to copy + paste from CNTK tutorials on GitHub and into Jupyter notebooks when scrolling is so slow.

DaveVoyles committed:
Further improvements to page scrolling

Instructions: https://support.mozilla.org/en-US/questions/981140

Type about:config in the URL and filter mousewheel.enable_pixel_scrolling and change the value as false
